### DISCUSSION TITLE
create next app > npm run dev > error Command failed with exit code 1

### DISCUSSION THREAD

**Discussion by SRD75** [vercel/next.js#48472] [Help] [answered]
### Summary

After running  `npx create-next-app@latest` then `npm run dev` I receive an error `Command failed with exit code 1`. 

### Additional information

```
    Operating System:
      Platform: win32
      Arch: ia32
      Version: Windows 10 Home
    Binaries:
      Node: 18.16.0
      npm: 9.5.1
      Yarn: 1.22.19
      pnpm: N/A
    Relevant packages:
      next: 13.3.0
      eslint-config-next: 13.3.0
      react: 18.2.0
      react-dom: 18.2.0
```

The project directory does not have any spaces in it.

Today, in the same project directory `npm run dev` produces no errors but the terminal process does not stay running, it ends, and the terminal is back at the prompt again.

**Comment by icyJoseph** [comment]
Hi,

Have you looked into whether or not the Node.js version you have installed is compatible with your system?

You can run: `node -p 'process.arch'` directly in your console, and that should match your system.

If it doesn't you gotta install a compatible Node.js version. Long shot, but it has helped quite a few people.

**Reply by SRD75** [reply]
Hi, thanks for replying.

`node -p 'process.arch'` produces `ia32`, which seems okay as my laptop's CPU is Intel. Windows is 64-bit.

**Reply by icyJoseph** [reply, marked as answer]
Mm I am not sure why this fails though, but could you install node for 64 bits then? (Isn't ia32 a 32 bits architecture?)

**Reply by SRD75** [reply]
That did it. I uninstalled the current node, downloaded and installed `node-v18.16.0-x64`, `node -p 'process.arch'` gives `x64`, and `npm run dev` builds the local client and server successfully at `http://localhost:3000`.

THANK YOU

**Reply by ndungtse** [reply]
> Mm I am not sure why this fails though, but could you install node for 64 bits then? (Isn't ia32 a 32 bits architecture?)

I also had the same problem but this approach works 😊😊!!. THANK YOU!!

**Reply by h1bba** [reply]
> Mm I am not sure why this fails though, but could you install node for 64 bits then? (Isn't ia32 a 32 bits architecture?)

thank you so much

**Comment by alamenai** [comment]
I faced the same issue but with pnpm and I resolved it by `pnpm update` after deleting the `node_modules` and clearing the cache.

**Reply by tcarvalho-mobinteg** [reply]
Also worked for me using npm on Pop!_OS 22.04, node v20.12.2
